examples/pbfhr/multi_pt/prt_2ref/input_10s.py

```diff
@@ -0,0 +1,206 @@
+''' changed from the initial input file
+to represent a simplified model for the FHR core
+
+The simulation has 3 stages:
+- initial temperature without feedback
+- turn on feedback
+- turn on external reactivity
+'''
+from utilities.ur import units
+import th_component as th
+import math
+from materials.material import Material
+from density_model import DensityModel
+from convective_model import ConvectiveModel
```

Comment:
I'm concerned that previous input files, missing this convective model improvement, will fail where they have previously used constant h. If so, this PR should include updates to all currently maintained input files, accordingly.